bigbluebutton-html5/imports/ui/components/meeting-ended/service.ts

```diff
@@ -95,9 +95,29 @@ export const setLearningDashboardCookie = (accessToken: string, mId: string, lea
   return false;
 };
 
+export const allowRedirectToLogoutURL = (logoutURL: string) => {
+  const ALLOW_DEFAULT_LOGOUT_URL = window.meetingClientSettings.public.app.allowDefaultLogoutUrl;
+  const protocolPattern = /^((http|https):\/\/)/;
+
+  if (logoutURL) {
+    // default logoutURL
+    // compare only the host to ignore protocols
+    const urlWithoutProtocolForAuthLogout = logoutURL.replace(protocolPattern, '');
+    const urlWithoutProtocolForLocationOrigin = window.location.origin.replace(protocolPattern, '');
```

Comment:
Every time we have `window.location`, this raises concerns about proxy cluster support. Anything we should change here?

Reply:
I'm not sure about it, but based on previous versions it should be ok (this method is the same that was being used until 2.7, since #10615)